Sources/PointFree/LaunchSignup/LaunchSignup.swift

```diff
@@ -25,6 +29,34 @@ private func airtableStuff<I>(_ conn: Conn<I, String>) -> IO<Conn<I, Either<Prel
   return result.map { conn.map(const($0)) }
 }
 
+func notifyUsOfNewSignup<I>(_ conn: Conn<I, String>) -> IO<Conn<I, String>> {
+  return IO {
+
+    // Fire-and-forget to notify us that someone signed up
+    _ = sendEmail(
+      from: "Point-Free <brandon@pointfree.co>",
+      to: ["mbw234@gmail.com", "stephen.celis@gmail.com"],
```

Comment:
will change this to `support@pointfree.co` once we get our forwarding figured out